airbyte-integrations/connectors/source-faker/source_faker/streams.py

```diff
@@ -4,21 +4,37 @@
 
 import datetime
 import os
+from multiprocessing import Pool, current_process
 from typing import Any, Dict, Iterable, Mapping, Optional
 
-from airbyte_cdk.models import AirbyteEstimateTraceMessage, AirbyteTraceMessage, EstimateType, TraceType
+from airbyte_cdk.models import AirbyteRecordMessage, Type
 from airbyte_cdk.sources.streams import IncrementalMixin, Stream
 from mimesis import Datetime, Numeric, Person
 from mimesis.locales import Locale
 
-from .utils import format_airbyte_time, read_json
+from .airbyte_message_with_cached_json import AirbyteMessageWithCachedJSON
+from .utils import format_airbyte_time, generate_estimate, now_millis, read_json
+
+
+class FakerMultithreaded:
```

Comment:
nitpick but nothing about this class (I think?) does any threading, so it was a little confusing to see it had `multithreaded` in the name